.rspec

```diff
@@ -0,0 +1 @@
+--require spec_helper
```

Comment:
I prefer to have the spec_helper explicit rather than implicitly hidden in the .rspec file, can you please adjust this? Thanks!